lib/java/com/google/android/material/appbar/res/values/attrs.xml

```diff
@@ -272,6 +272,35 @@
          should be centered horizontally within the toolbar. Default is false.
     -->
     <attr name="subtitleCentered" format="boolean" />
+    <!-- adjust view bounds for logo's imageView. Default is false.   -->
+    <attr name="logoAdjustViewBounds" format="boolean" />
```

Comment:
Update the doc and public.xml as well.

Reply:
Where doc?

Reply:
https://github.com/material-components/material-components-android/blob/master/docs/components/TopAppBar.md